packages/ff-bench/ff-bench.0.6.2/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Benchmark library for finite fields over the package ff-sig"
+description: "Benchmark library for finite fields over the package ff-sig"
```

Comment:
the two seem a bit redundant. The `synopsis` looks good but the description is expected to dive a bit deeper. If both are equal then you might want to remove the `description` field as only the synopsis is strictly necessary.

Reply:
(same thing for the other packages)

Reply:
Done upstream, removing the description field. I change also here.